new_project/approval_template.md

```diff
@@ -0,0 +1,23 @@
+<!---
```

Comment:
This is probably my ignorance, but is this just a comment that only shows up when they are opening the issue? Or why doesn't it render when you look at this file?

Reply:
It is just for when you are editing the file. It is for providing instructions to the author that the reader doesn't need to know about.